### PR TITLE
Code to print fibonacci sequence in Javascript

### PR DESCRIPTION
**Pull request by rahulbasrani**
## Description
Onboarding Exercise 1 for day 1, print fibanocci sequence upto n terms

## Database schema changes
_None_

## Tests
_Describe the automated and manual tests you performed to verify it works._

### Automated test cases added
- _None_

### Manual test cases run
- Ran index.js locally, Working as expected.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var a, b, result; a = 0;
+b = 1;
+result = b;
+var b = "jason";
+for (var i = 1; i < 100; i++) {
+	console.log(result);
+	result = a + b;
+	a = b;
+	b = result;
+}
```
Comment (akashlakhera) [inline index.js]: Please add newline at end

**Comment by akashlakhera** [comment]
Also the code is not working as expected (infinite loop).
Please check.

File: index.js (hunk 1 above)
Comment (jjalan) [reply index.js]: Use let instead of var.

**Comment by jjalan** [comment]
Also,

1. Incorrect PR template
2. Please update PR title
3. fill out PR template.

Here is an example that one of the other team did - https://github.com/balwant-jtc/eng-onboarding/pull/1

File: index.js (hunk 1 above)
Comment (rahulbasrani) [reply index.js]: ok

**Comment by rahulbasrani** [comment]
@akashlakhera @jjalan I have modified PR as desired,
if any suggestions please provide.

**Comment by akashlakhera** [comment]
LGTM

**Review by akashlakhera — APPROVED** [review]

File: index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+let a, b, result; a = 0;
+b = 1;
+result = b;
+for (let i = 1; i < 10; i++) {
+	console.log(result);
```
Comment (jjalan) [inline index.js]: Use 2 spacing instead of 4.

Comment (rahulbasrani) [reply index.js]: okk.

**Review by jjalan — APPROVED** [review]

**Pull request merged**